HackYourFuture-CPH/kp-toolbox-fp-class21
# Feature/fp21-24-tools-grid-area-frontend

# Description

This pull request  is about tool item and tool grid, it contains
1. Backend rout to provide api.
2.  Migration file and seed to provide database(not the final database, but a sample contain that we needed)
3. ToolItem and ToolGrid Component which contain the frontend and UI, with responsive design in the CSS.

Fixes # (issue)

# How to test?

1. Pull the files, and add .env file to the client folder and add (REACT_APP_API_BASE_URL="http://localhost:5001") to it.
2. run  “yarn knex migrate:latest <your create_tools file name without .js>" in the migrations folder
3. open seeds directory ,run “yarn knex seed: run 01_tools” to add the data from seed file.
4. run docker, frontend and backend servers.
5. Add <ToolsGrid/> to landing page, so You will be able to see the output in the browser.



# Checklist
